src/main/scala/mesosphere/marathon/core/launchqueue/impl/ReviveOffersActor.scala

```diff
@@ -55,6 +61,23 @@ class ReviveOffersActor(
     b.addAllRoles(roles.asJava)
     b.build
   }
+  private def requestWithResources(resources: Resources): Request = {
+    val b = Request.newBuilder;
+    b.addResources(
+      Resource.newBuilder.setName("cpus")
+        .setType(SCALAR)
+        .setScalar(Scalar.newBuilder().setValue(resources.cpus)))
+      .addResources(
+        Resource.newBuilder.setName("mem")
+          .setType(SCALAR)
+          .setScalar(Scalar.newBuilder().setValue(resources.mem)))
+      .addResources(
```

Comment:
FAR: add gpu as well since gpu is a "scarce" resource and we'll have the same issue for them